elide-datastore/elide-datastore-hibernate3/src/main/java/com/yahoo/elide/datastores/hibernate3/HibernateStore.java

```diff
@@ -21,14 +22,62 @@
  */
 public class HibernateStore implements DataStore {
     private final SessionFactory sessionFactory;
+    private final boolean isScrollEnabled;
+    private final ScrollMode scrollMode;
 
     /**
      * Initialize HibernateStore and dictionaries.
      *
      * @param aSessionFactory the a session factory
+     * @deprecated since 2.3.2. Use builder instead.
      */
+    @Deprecated
     public HibernateStore(SessionFactory aSessionFactory) {
         this.sessionFactory = aSessionFactory;
+        this.isScrollEnabled = true;
+        this.scrollMode = ScrollMode.FORWARD_ONLY;
+    }
+
+    /**
+     * Constructor.
+     *
+     * @param aSessionFactory Session factory
+     * @param isScrollEnabled Whether or not scrolling is enabled on driver
+     * @param scrollMode Scroll mode to use for scrolling driver
+     */
+    private HibernateStore(SessionFactory aSessionFactory, boolean isScrollEnabled, ScrollMode scrollMode) {
+        this.sessionFactory = aSessionFactory;
+        this.isScrollEnabled = isScrollEnabled;
+        this.scrollMode = scrollMode;
+    }
+
+    /**
+     * Builder object to configuration hibernate transaction.
+     */
+    public static class Builder {
+        private final SessionFactory sessionFactory;
+        private boolean isScrollEnabled;
+        private ScrollMode scrollMode;
+
+        public Builder(final SessionFactory sessionFactory) {
+            this.sessionFactory = sessionFactory;
+            this.isScrollEnabled = true;
+            this.scrollMode = ScrollMode.FORWARD_ONLY;
+        }
+
+        public Builder isScrollEnabled(final boolean isScrollEnabled) {
```

Comment:
i think method names that start with "is" are supposed to return boolean. I think you meant: `withScrollEnabled`